packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -115,6 +119,21 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
           : 20
         : this.props.margins?.left ?? 40,
     };
+    if (this.props.xAxisTitle !== undefined && this.props.xAxisTitle !== '') {
+      this.margins.bottom! = this.props.margins?.bottom ?? 70;
+    }
+    if (this.props.yAxisTitle !== undefined && this.props.yAxisTitle !== '') {
+      this.margins.left! = this._isRtl
+        ? this.props.margins?.right ?? this.props?.secondaryYScaleOptions
+          ? 80
+          : 40
+        : this.props.margins?.left ?? 80;
+      this.margins.right! = this._isRtl
+        ? this.props.margins?.left ?? 80
+        : this.props.margins?.right ?? this.props?.secondaryYScaleOptions
```

Comment:
This should be secondaryYAxistitle not secondaryYScaleOptions

Reply:
done